rel/emqx_conf.template.en.md

```diff
@@ -7,22 +7,27 @@ and a superset of JSON.
 EMQX configuration consists of two layers.
 From bottom up:
 
-1. Cluster configs: `$EMQX_NODE__DATA_DIR/configs/cluster.hocon`
-2. `emqx.conf` + `EMQX_` prefixed environment variables.<br/>
-   Changes in this layer require a full node restart to take effect.
+1. Cluster-synced configs: `$EMQX_NODE__DATA_DIR/configs/cluster.hocon`.
+2. Local node configs: `emqx.conf` + `EMQX_` prefixed environment variables.
 
+:::tip Tip
+Prior to v5.0.23 and e5.0.3, the cluster-synced configs are stored in
+`cluster-override.conf` which on top of the local configs.
+
+If upgrade from earlier version, as long as `cluster-override.conf` exists,
```

Comment:
```suggestion
If you upgrade from earlier version, as long as `cluster-override.conf` exists,
```

Reply:
changed to: If upgraded from an earlier version.